guides/common/modules/proc_running-ansible-roles-in-diff-mode.adoc

```diff
@@ -0,0 +1,16 @@
+[id="running-ansible-roles-in-diff-mode_{context}"]
```

Comment:
```suggestion
[id="running-ansible-roles-in-diff-mode"]
```

For new/changed anchors, we omit the context attribute unless explicitely necessary.